src/main/java/com/iota/iri/controllers/TipsViewModel.java

```diff
@@ -1,27 +1,22 @@
 package com.iota.iri.controllers;
 
-import com.iota.iri.model.Approvee;
 import com.iota.iri.model.Hash;
-import com.iota.iri.model.Transaction;
-import com.iota.iri.storage.Indexable;
-import com.iota.iri.storage.Tangle;
-import org.apache.commons.lang3.ArrayUtils;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.security.SecureRandom;
 import java.util.*;
 import java.util.concurrent.ExecutionException;
-import java.util.stream.Collectors;
 
 /**
  * Created by paul on 3/14/17 for iri-testnet.
  */
 public class TipsViewModel {
     final Logger log = LoggerFactory.getLogger(TipsViewModel.class);
 
-    private FifoHashCache tips = new FifoHashCache(5000);
-    private FifoHashCache solidTips = new FifoHashCache(5000);
+    public static final int MAX_TIPS = 5000;
```

Comment:
Why not configurable?

Reply:
this class don't have access to configuration.
as configuration will be redone shortly, it's not worth changing the interface imo
(see https://github.com/iotaledger/iri/pull/404)